### DISCUSSION TITLE
Frage: HM-LC-BI1PBU-FM mit Schalter?

### DISCUSSION THREAD

**Discussion by flohei** [thkl/hap-homematic#302] [General]
Hallo!

Ich habe einen HM-LC-BI1PBU-FM in Verwendung. Der taucht in Home mit einem "Rollo-Schalter" und einem gewöhnlichen Switch auf. Letzterer scheint aber keine Funktion zu haben. Nach der Aktivierung springt er einfach wieder zurück. Wofür ist der gedacht?

Danke!

**Comment by thkl** [comment]
das sollte zu einem Rollo werden:

![image](https://user-images.githubusercontent.com/884355/103222489-ade44280-4924-11eb-8e55-ff897df78844.png)

Und das Rollo lässt sich durch hoch und runter schieben des Reglers bedienen:

![image](https://user-images.githubusercontent.com/884355/103222758-061b4480-4925-11eb-895f-fe7f758e2bab.png)

**Comment by flohei** [comment]
Das habe ich schon auch. Nur eben zusätzlich auch den Schalter. Das Rollo habe ich umbenannt. ![image](https://user-images.githubusercontent.com/192587/103224855-dd954980-4928-11eb-8d4e-ac129f3e429f.png)

**Comment by thkl** [comment]
ich habe nicht die geringste Ahnung, wo der her kommen könnte. In den Eigenschaften des HomeKit Gerätes steht in der Seriennummer die Seriennummer des Gerätes und bei Model was es für ein Channeltyp ist, der verwendet wurde. Vielleicht hilft das bei der Suche.

**Comment by flohei** [comment]
Okay, ich hab's gefunden. Ich habe wohl testhalber unter "Special Devices" mal was eingetragen. Da kam das her.
